app/Constants/Messages.php

```diff
@@ -0,0 +1,9 @@
+<?php
+declare(strict_types = 1);
+
+namespace App\Constants;
+
+class Messages
+{
+    const MESSAGE = 'message';
```

Comment:
в php 7 появились области видимости для констант
```suggestion
    public const MESSAGE = 'message';
```